static/js/components/forms/ProfileFormFields.js

```diff
@@ -66,8 +61,9 @@ export const legalAddressValidation = yup.object().shape({
   })
 })
 
-export const passwordValidation = yup.object().shape({
-  password: newPasswordFieldValidation
+export const newLegalAddressValidation = yup.object().shape({
+  password: newPasswordFieldValidation,
+  username: usernameFieldValidation
 })
```

Comment:
`newLegalAddressValidation` doesn't seem to match the configuration values, did you mean `newAccountValidation`?

Reply:
Good point. I was following the existing naming convention without really thinking about it